packages/render/src/types.ts

```diff
@@ -74,7 +78,7 @@ export type OnConnectWalletFunc = () => void;
 export type SignFrameActionFunc<
   TSignerStorageType = Record<string, unknown>,
   TFrameActionBodyType extends FrameActionBodyPayload = FrameActionBodyPayload,
-  TFrameContextType extends FrameContext = FarcasterFrameContext,
```

Comment:
We should probably create a prettier config that doesn't remove those commas. Introduces unnecessary changes in diff.